### PR TITLE
Implement get_patrol_observations task

### PR DESCRIPTION
**Pull request by Yun-Wu**
- **Implement get_patrol_observations**
- **e2e test**

### CODE REVIEW

**Comment by Yun-Wu** [comment]
[#42](https://github.com/wildlife-dynamics/ecoscope-workflows/issues/42)

**Review by cisaacstern — COMMENTED** [review]
Thanks so much Yun! Overall looks awesome!

Let's see if we can get https://github.com/wildlife-dynamics/ecoscope-workflows/pull/59 done first, so we can add a test for this, and then I'll do a full review. 🙏

File: ecoscope_workflows/decorators.py [hunk 1]
```diff
@@ -118,7 +118,11 @@ def __call__(self, *args, **kwargs):
                 "and post- call behavior is only modified when `self.validate=True`."
             )
         return (
-            validate_call(func_copy, validate_return=True)(*args, **kwargs)
+            validate_call(
+                func_copy,
+                validate_return=True,
+                config={"arbitrary_types_allowed": True},
```
Comment (cisaacstern) [inline ecoscope_workflows/decorators.py:124]: I think this should be fine! In fact, I'm a little surprised we didn't have it already 😄

**Comment by cisaacstern** [comment]
@Yun-Wu, @atmorling's work in #59 should provide a good template for writing a unit test for this. Would you mind adding that test before we merge? Thanks!

**Comment by Yun-Wu** [comment]
Added unittest. Also updated `get-subjectgroup-observations.example-return.parquet` to fix test `test_get_subject_group_observations`

**Pull request merged**